server/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -310,6 +323,8 @@ public class SearchService extends AbstractLifecycleComponent implements IndexEv
 
     private volatile boolean enableRewriteAggsToFilterByFilter;
 
+    private volatile ByteSizeValue memoryAccountingBufferSize;
```

Comment:
Whether or not we make this constant, why not make it a `long` and save all those to byte conversions?

Reply:
Yep good shout.